lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -133,11 +142,15 @@ public function purge()
         }
 
         $connection = $this->em->getConnection();
+        $filterExpr = $connection->getConfiguration()->getFilterSchemaAssetsExpression();
+        $efe = empty($filterExpr);
```

Comment:
Ok, i changed the variable name and added tests for exclusion using the filter expression or the explicit exclusion list.